epick_driver/src/default_driver.cpp

```diff
@@ -74,7 +74,10 @@ namespace epick_driver
 {
 const auto kLogger = rclcpp::get_logger("DefaultDriver");
 
-constexpr float kAtmosphericPressure = 100;  // kPa.
+constexpr auto kAtmosphericPressure = 100.0f;  // kPa.
```

Comment:
Would anything change if someone forgot to specify the type in the literal when changing the constant value?

Reply:
Nothing would change because later in the code it is rounded and clamped even if it was a double.